resources/js/Pages/Home.vue

```diff
@@ -42,18 +43,51 @@
             user: Object,
         },
         methods: {
+            checkEmpty() {
+                if( !this.form.itemsInput ) {
+                    this.error = {
+                        type: 'warning',
+                        message: 'item-form-error-message-empty'
+                    };
+                    return false;
+                }
+                return true;
+            },
+            validate() {
```

Comment:
So I think this method could probably be split into two or three: one to deserialize the IDs into an array, one to perform checks and set the error, and another one to serialize the IDs into the pipe separated format.

That way not only are you ensuring each function has its own responsibility to take care of, but you are also decoupling the validation and sanitization logic from one another.